docs/docs/guides/create_dagger.md

```diff
@@ -6,13 +6,79 @@ This page contains how-to guides for creating a Dagger job and configure it.
 
 Dagger is a stream processing framework built with Apache Flink to process/aggregate/transform protobuf data. To run a dagger in any environment you need to have the following things set up beforehand.
 
-#### `JDK and Gradle`
+### `JDK and Gradle`
 
 - Java 1.8 and gradle(5+) need to be installed to run in local mode. Follow this [link](https://www.oracle.com/in/java/technologies/javase/javase-jdk8-downloads.html) to download Java-1.8 in your setup and [this](https://gradle.org/install/) to set up gradle.
 
-#### `Kafka Cluster`
+### `A Source`
 
-- Dagger use [Kafka](https://kafka.apache.org/) as the source of Data. So you need to set up Kafka(1.0+) either in a local or clustered environment. Follow this [quick start](https://kafka.apache.org/quickstart) to set up Kafka in the local machine. If you have a clustered Kafka you can configure it to use in Dagger directly.
+Dagger currently supports 3 kinds of Data Sources. Here are the requirements for each:
+
+##### `KAFKA_SOURCE` and `KAFKA_CONSUMER`
+
+Both these sources use [Kafka](https://kafka.apache.org/) as the source of data. So you need to set up Kafka(1.0+) either 
+in a local or clustered environment. Follow this [quick start](https://kafka.apache.org/quickstart) to set up Kafka in 
+the local machine. If you have a clustered Kafka you can configure it to use in Dagger directly.
+
+##### `PARQUET_SOURCE`
+
+This source uses Parquet files as the source of data. The parquet files can be either hourly partitioned, such as
+```text
+root_folder
+    - booking_log
+        - dt=2022-02-05
+            - hr=09
+                * g6agdasgd6asdgvadhsaasd829ajs.parquet
+                * . . . (more parquet files)
+            - (...more hour folders)
+        - (... more date folders)
+
+```
+
+or data partitioned, such as:
```

Comment:
small typo here, I believe we went date partitioned here.

Reply:
Fixed via commit https://github.com/odpf/dagger/pull/153/commits/31f5ec3aacaf808fc27ad70764169faddb77287d